scipioninstaller/installer.py

```diff
@@ -132,23 +141,25 @@ def getRepoInstallCommand(scipionHome, repoName, useHttps, organization='scipion
 
 
 def getInstallationCmd(scipionHome, dev, args):
+
+    cmd = cmdfy("cd %s" % scipionHome)
+    cmd += cmdfy("mkdir -p software/lib")
+    cmd += cmdfy("mkdir -p software/bindings")
+    cmd += cmdfy("mkdir -p software/em")
+
     if dev:
         useHttps = args.httpsClone
         noXmipp = args.noXmipp
```

Comment:
I would put `noXmipp = args.noXmipp` line out of the IF and, then, installing Xmipp by default also for non dev version.

Reply:
BUt for that we need a final xmipp-20.5.tgz?

Reply:
Do you mean installp -p scipion-em-xmipp ?

Reply:
Yes, I mean that. And, yes, for that we net a released version of Xmipp. I'm releasing an alpha version: 20.07a0 (I bet for July, who join me ;-p?), both for binaries and pypi.